pkg/sfu/buffer/buffer.go

```diff
@@ -382,11 +382,14 @@ func (b *Buffer) calc(pkt []byte, arrivalTime int64) {
 	}
 	b.lastTransit = transit
 
-	if b.audioLevel {
+	if b.audioLevel && headPkt {
```

Comment:
In practice this is not going to be a problem as audio packets are small.

But, to ensure protocol compliance, I think we should remove the `headPkt` check here. Assume that an audio frame is split into two RTP packets and only the first one (the one which is not `headPkt`) has the ssrc-audio-level extension and the second one (one with `headPkt`) does not have it. Then this code will not get the ssrc-audio-level. There is already a check below for 0 duration. That should take care of the case where a packet has been split into two and both have ssrc-audio-level as both those should have same RTP timestamp. That will take care of avoiding double counting.

Reply:
Never see a audio frame split to 2 packets, and just first one has audio level. Does it exist only in theory?

Reply:
Only in theory Jie. I have not seen it in practice. But, just in case some crazy implementation does it, let's not get bitten by it :-)